modules/persistent-storage-csi-cloning-overview.adoc

```diff
@@ -19,6 +19,8 @@ No new API objects are required for cloning. The existing `dataSource` field in
 By default, {product-title} supports CSI volume cloning with these limitations:
 
 * The destination persistent volume claim (PVC) must exist in the same namespace as the source PVC.
-* The source and destination storage class must be the same.
+* Cloning is supported with a different Storage Class.
+** Destination volume can be the same or a different storage class as the source.
```

Comment:
This sentence confused me. It isnt supposed to be 
```suggestion
** Destination volume can be the same for a different storage class as the source.
```

Reply:
Addressed